base-template/about-base-template.md

```diff
@@ -1,8 +1,21 @@
 # About Base Template
 
+{Template author tip:
+
+* This about-base-template captures the structure and common elements that all downstream about-xxx-templates should follow.
+* It provides guidance for both template authors as well as document authors.
+* “Template author and template reviewer tips” should be removed from the final about-{doctype}-template.md. Other tips should remain.
+* Tweak other tips to align with the doctype you are writing for.
```

Comment:
```suggestion
* Adjust other tips to align with the doctype you are writing for.
```
or "modify"
Tweak has drug connotations in the USA

Reply:
Thanks, fixed.